Makefile

```diff
@@ -45,6 +59,14 @@ deploy-cert-manager:
 		--set installCRDs=true \
 		--set extraArgs[0]="--enable-certificate-owner-ref=true""
 
+# Run this only if your minikube does not have cert-manager already deployed
+deploy-cert-manager-minikube:
+	$(HELM_CMD_MINI) "helm repo add jetstack https://charts.jetstack.io && helm upgrade -i --wait cert-manager --namespace cert-manager jetstack/cert-manager --version v1.0.1 \
+		--create-namespace \
+		--kube-context minikube \
+		--set installCRDs=true \
+		--set extraArgs[0]="--enable-certificate-owner-ref=true""
```

Comment:
This duplicates existing functionality. I don't understand why the existing commands aren't working for your use case.

Officially, we support testing in `kind`. That works like this:
```
make kind
make deploy-cert-manager deploy-ingress-nginx
make kind-load-konk deploy-konk-operator
```

The equivalent should work for minikube:
```
minkube start
make deploy-cert-manager deploy-ingress-nginx
make kind-load-konk deploy-konk-operator
```
What happens when you run these commands?

Reply:
after `minikube start`, when running `make deploy-cert-manager`, the following errors came out

```
Error: Kubernetes cluster unreachable: invalid configuration: [unable to read client-cert /Users/hdu/.minikube/client.crt for minikube due to open /Users/hdu/.minikube/client.crt: no such file or directory, unable to read client-key /Users/hdu/.minikube/client.key for minikube due to open /Users/hdu/.minikube/client.key: no such file or directory, unable to read certificate-authority /Users/hdu/.minikube/ca.crt for minikube due to open /Users/hdu/.minikube/ca.crt: no such file or directory]
make: *** [deploy-cert-manager] Error 1
```

Reply:
embed certificates into the kubeconfig, some faq docs on this here: https://github.com/getporter/porter/pull/664/files#diff-83cea8d3073291cd4afd5c393ef0060b7fdc246e13eebf62029399d3556b9e92

Reply:
We use a docker container to call helm which is what is causing you issues. You could turn it off `DOCKER_RUNNER=helm make deploy-konk-operator`